platform-hub-web/src/app/projects/projects-detail.component.js

```diff
@@ -283,4 +284,40 @@ function ProjectsDetailController($rootScope, $q, $mdDialog, $state, roleChecker
           });
       });
   }
+
+  function offboardAndRemove(membership, targetEvent) {
+    if (!ctrl.isAdmin && !ctrl.isProjectManager) {
+      return;
+    }
+
+    const confirm = $mdDialog.confirm()
+      .title('Are you sure?')
+      .textContent('This will offboard this person from GitHub and remove them from the project. Continue with caution as this will remove their access from repositories, etc.')
+      .ariaLabel('Confirm offboarding from GitHub and removal of team member from project')
```

Comment:
maybe: "This will remove the person from the github organisation as well as this team"